### ISSUE TITLE
Doc improvement idea: explicit required GITHUB_TOKEN permissions documented

### ISSUE THREAD

**Issue by stutommi** [google-github-actions/release-please-action#620]
### TL;DR

When dealing with with GITHUB_TOKEN permissions in workflow yml for OIDC, release-please breaks down with multiple `Resource not accessible by integration`-errors. without explicit permissions in yml-file. the action works fine.

Would be great if the exact permission-requirements would be documented in the main README.md in this repo. :)

### Detailed design

Using OIDC within an workflow, one has to put explicit permissions:
```yml
permissions:
      id-token: write
      contents: read
```
When this is done in the same workflow where release-please would be defined as following:
```yml
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: release-please-action
```
The result is below
_______________
✔ Looking for latest release on branch: main with prefix: release-please-action
Error: release-please failed: Request failed due to following response errors:
 - Resource not accessible by integration
 - Resource not accessible by integration
 - Resource not accessible by integration
 - Resource not accessible by integration
 - Resource not accessible by integration
 - Resource not accessible by integration
 ________________

**Comment by stutommi** [comment]
Continuing at this, on my own case it was possible (and probably) to change the permissions at job level, where release-please was not at the same job. This fixed my problem, but having explicit knowledge on GITHUB_TOKEN permission requrements would still be good to have in my opinion.

**Comment by bcoe** [comment]
@stutommi would happily add this to the README, seems like a good suggestion. Did you want to make the update, what permissions did you end up enabling for your workflow?
